### PR TITLE
Fixing bug described in #9206

### PR DESCRIPTION
**Pull request by jack-webster**
As per Taylor's comments on #9217

### CODE REVIEW

File: src/Illuminate/Routing/RouteCollection.php [hunk 1]
```diff
@@ -106,7 +106,7 @@ protected function addLookups($route)
      */
     protected function addToActionList($action, $route)
     {
-        $this->actionList[$action['controller']] = $route;
+        $this->actionList[trim($action['controller'],'\\')] = $route;
```
Comment (GrahamCampbell) [inline src/Illuminate/Routing/RouteCollection.php]: cs

Comment (GrahamCampbell) [inline src/Illuminate/Routing/RouteCollection.php]: Also, why two identical slashes? You don't need to escape the 2nd slash.

**Comment by GrahamCampbell** [comment]
Please squash to one commit and send to the correct branhc.

File: src/Illuminate/Routing/RouteCollection.php (hunk 1 above)
Comment (jack-webster) [inline src/Illuminate/Routing/RouteCollection.php]: You do, otherwise it escapes the closing ' and doesn't close the string.

**Pull request closed without merging**